test/models/API1811/DocumentDetailsSpec.scala

```diff
@@ -31,7 +31,7 @@ class DocumentDetailsSpec extends SpecBase {
       }
 
       "minimum fields are present" in {
-        Json.obj("lineItemDetails" -> Json.arr("")).as[DocumentDetails] shouldBe emptyDocumentDetails
+        Json.obj("lineItemDetails" -> Json.arr(""), "documentPenaltyTotals" -> Json.arr("")).as[DocumentDetails] shouldBe emptyDocumentDetails
```

Comment:
Minimum fields would not include `documentPenaltyTotals`. Technically it wouldn't include the line items either but I think these have been added to avoid the exception we would throw if there were no line items present (there is a separate test for this)